lib/Runtime/Language/TaggedInt.inl

```diff
@@ -206,7 +206,23 @@ namespace Js
     }
 #endif
 
+    // Returns true when the provided args only contains TaggedInts. Note that
+    // iteration starts from 0 (to account for 'this' at index 0).
+    // ** Is this too large of a function to inline?
```

Comment:
@LouisLaf , @pleath -- fyi, most of the comments in this first round are questions about assumptions I made that I need verified